core/src/leader_slot_banking_stage_metrics.rs

```diff
@@ -104,6 +104,11 @@ struct LeaderSlotPacketCountMetrics {
     // then hit the age limit after failing to be comitted.
     executed_transactions_failed_commit_count: u64,
 
+    // total number of transactions that were excluded from the block because there were concurrent write locks active.
+    // These transactions are added back to the buffered queue and are already counted in
+    // `self.retrayble_errored_transaction_count`.
```

Comment:
thanks for clarifying this counter is a subset of `retrayble_errored_transaction_count`